tests/aws/services/secretsmanager/test_secretsmanager.py

```diff
@@ -2395,6 +2396,56 @@ def test_get_secret_value_errors(self, aws_client, create_secret, sm_snapshot):
             )
         sm_snapshot.match("mismatch_version_id_and_stage", exc.value.response)
 
+    @markers.aws.validated
+    def test_get_secret_value_to_fix_11319(
```

Comment:
```suggestion
    def test_get_binary_secret_value(
```
nit: Don't need to specify the issue in here